### DISCUSSION TITLE
Réception des données du formulaire transmises à notre système

### DISCUSSION THREAD

**Discussion by IhsaneRQ** [MTESSDev/FRW#28] [Annonces]
Bonjour,

Nous voulons créer dans notre API une nouvelle ressource pour la réception des données du formulaire transmises à notre système, nous avons besoin de savoir quelles sont les entêtes, les codes de retour prévus, le verbe http et la structure des données (un exemple serait apprécié) pour la réception du formulaire?

Merci d'avance

**Comment by anisite** [comment]
Le travail était déjà en cours, j'ai ouvert un issue pour gérer l'état, je vous tiens au courant #30

**Reply by IhsaneRQ** [reply]
Merci beaucoup pour le retour rapide

**Reply by anisite** [reply]
bonjour, nous avons du faire de petits changements pour rendre plus simple la documentation et l'utilisation de toute la partie "transmission" je suis dans les derniers ajustements. On va déployer et vous demander de "tester" notre documentation.

**Comment by anisite** [comment]
Bonsoir, nous serions prêt à faire tester le premier bout de la doc. il manque la liste de toutes les variables possibles pour un http_client mais je vais les ajouter prochainement, ça ne vous empêche pas de tester votre premier fichier de transmission pour vous familiariser avec nos outils de débug et la structure de données pour l'API, la documentation est ici : https://github.com/MTESSDev/FRW/blob/main/Documentation/creer-premier-fichier-transmission.md

tennez-moi au courant de vos impression, on va bonifier le tout à mesure :)

**Reply by IhsaneRQ** [reply]
Bonjour, Ok parfait merci et bonne journée

**Reply by IhsaneRQ** [reply]
Bonjour,
Avez-vous svp de nouvelle concernant la suite de la documentation?

Merci et bonne journée

**Reply by anisite** [reply]
Bonjour, ça été déposé ce matin et disponible ici [Documentation/fichiers-transmission.md](https://github.com/MTESSDev/FRW/blob/main/Documentation/fichiers-transmission.md) (tout en bas de la page) la liste des variables explique en gros ce que {{{Json .}}} va "dumper" il est possible de bâtir en sortie son propre contenu Json personnalisé pour votre service de réception, mais si vous êtes capables de le garde "vanille" c'est encore mieux!
